tests/NovaTestGeneratorTest.php

```diff
@@ -28,13 +28,13 @@ public function testCreateNovaTestsResourceNotExists()
         $mock = $this->mockClassExistsFunction();
 
         $this->expectException(ClassNotExistsException::class);
-        $this->expectExceptionMessage("Cannot create NovaPostTest cause Post Nova resource does not exist. Create Post Nova resource.");
+        $this->expectExceptionMessage("Cannot create NovaSomePostTest cause SomePost Nova resource does not exist. Create SomePost Nova resource.");
 
         $generatorMock = $this->getGeneratorMockForNonExistingNovaResource();
 
         try {
             $generatorMock
-                ->setModel('Post')
+                ->setModel('SomePost')
```

Comment:
let's use more concrete example for all tests

```suggestion
                ->setModel('WelcomeBonus')
```